webapp/channels/webpack.config.js

```diff
@@ -141,6 +139,7 @@ var config = {
             crypto: require.resolve('crypto-browserify'),
             stream: require.resolve('stream-browserify'),
             buffer: require.resolve('buffer/'),
+            process: require.resolve('process/browser.js'),
```

Comment:
why do we need process?

Reply:
I forget why I did this actually. I'll double check if this one is needed or if I just needed to change it below where it's passed into `webpack.ProvidePlugin`

Reply:
I think @pvev actually added that in his PR, but it showed up here since mine is based off of his and needed to be rebased when you reviewed this. I removed it though since it doesn't seem to be needed because we never `import from 'process'` in the web app code